src/components/Header.tsx

```diff
@@ -1,21 +1,22 @@
+import { Link, useLocation } from "react-router";
 import { Button } from "./ui/Button.tsx";
 import { Search } from "./ui/Search.tsx";
 
```

Comment:
Nit: We don't need to add file extensions.

```suggestion
import { Button } from "./ui/Button";
import { Search } from "./ui/Search";
```

Reply:
I have updated imports for `Button` and `Search` component (https://github.com/krushna-simform/filmfusion/pull/3/commits/504debe0ef184a81c7551dc54dfc76adf2ce7b5d)